Model/Repository.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Ziffity\Task\Model;
+
+use Ziffity\Task\Api\RepositoryInterface;
+use Ziffity\Task\Api\Data;
+use Ziffity\Task\Model\ResourceModel\Custom as ResourceCustomCart;
+use Ziffity\Task\Model\ResourceModel\Custom\CollectionFactory as CustomCartCollectionFactory;
+use Magento\Framework\Api\DataObjectHelper;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessorInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Exception\CouldNotDeleteException;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Reflection\DataObjectProcessor;
+use Ziffity\Task\Api\Data\CustomInterfaceFactory;
+
+class Repository implements RepositoryInterface
+{
+    protected $resource;
+    protected $customCartFactory;
+    protected $customCartCollectionFactory;
+    protected $searchResultsFactory;
+    protected $dataObjectHelper;
+    protected $dataObjectProcessor;
+    protected $dataCustomCartFactory;
+    private $collectionProcessor;
+    public function __construct(
+        ResourceCustomCart $resource,
+        CustomFactory $customCartFactory,
+        CustomInterfaceFactory $dataCustomCartFactory,
+        CustomCartCollectionFactory $customCartCollectionFactory,
+        Data\CartInterfaceFactory $searchResultsFactory,
```

Comment:
@NandhiniKarthi Already told this and you're doing it partially for the nth time